ambry-api/src/main/java/com.github.ambry/rest/RestUtils.java

```diff
@@ -114,6 +115,11 @@
      */
     public final static String OWNER_ID = "x-ambry-owner-id";
     /**
+     * optional in request; defines options while getting the blob and is optional support in a
+     * {@link BlobStorageService}. Valid values are available in {@link GetOptions}. Defaults to {@link GetOptions#None}
+     */
+    public final static String GET_OPTIONS = "x-ambry-get-options";
```

Comment:
How does this header allow multiple options? (as in why is this not called `x-ambry-get-option`)?